src/common/actions/repositories.js

```diff
@@ -5,8 +5,7 @@ import { conf } from '../helpers/config';
 
 const BASE_URL = conf.get('BASE_URL');
 
-export const FETCH_REPOSITORIES = 'FETCH_REPOSITORIES';
-export const FETCH_REPOSITORIES_SUCCESS = 'FETCH_REPOSITORIES_SUCCESS';
```

Comment:
Sorry, my message is really about the future not this PR, so error is still it's own action here.

I would argue that the logic is similar and not more, but I'm not speaking from experience. For example in place of a case of ERROR and SUCCESS you're testing for properties on the payload, which is the same logic but different.